### ISSUE TITLE
Converging AP and AR on Faster RCNN faster_rcnn_resnet50_fpn

### ISSUE THREAD

**Issue by mkport1** [pytorch/vision#1137]
Hello, 

I'm trying to implement the torchvision official tutorial for finetuning object detection. 
I have 8 classes, and about 4000 images that are labeled. 
The metrics at the end of each epoch, seems to be converging at a certain point to values lower that 0.5 (between 0.4 and 0.45 for recall and 0.2 to 0.3 for recall). 
On tensorboard my results look like this : 
<img width="327" alt="res2" src="https://user-images.githubusercontent.com/44587986/61472130-21d69900-a984-11e9-95d4-419a8bc73adf.png">
<img width="327" alt="res1" src="https://user-images.githubusercontent.com/44587986/61472235-55192800-a984-11e9-9f10-40ec3dfe8a63.png">

Blue curve is a tryout with a learning rate of 0.01 in the SGD optimizer
Pink curve is with a learning rate of 0.001

It looks like in both cases it stops learning very early. 
The weirdest part is that when I'm trying to evaluate on the training dataloader instead of the test, to see if it is overfitting, i'm getting low results as well (0.1 for AP and 0.3 for recall). 

Does anyone have an idea why I get such low results ? 
My hyperparameters (with learning rate 0.01 or 0.001) are the following : 
 momentum=0.9, weight_decay=0.0005 in the optimizer, batch_size of 16 on the train, 1 on the test. 
The dataloader test size is 50 as in the torchvision tutorial, because I noticed increasing the size of the test dataset was giving worse performance. 
My model is the following : 
 model = torchvision.models.detection.fasterrcnn_resnet50_fpn(pretrained=True)
    # get features

    in_features = model.roi_heads.box_predictor.cls_score.in_features
    num_classes = 9
    # last head replaced with the new head that will be trained

    model.roi_heads.box_predictor = FastRCNNPredictor(in_features, num_classes)

    dataset = Custom("CustomDatasetFolder", get_transform(train=True))
    dataset_test = Custom("CustomDatasetFolder", get_transform(train=False))

    # split the dataset in train and test set
    torch.manual_seed(1)
    indices = torch.randperm(len(dataset)).tolist()
    dataset = torch.utils.data.Subset(dataset, indices[:-50])
    dataset_test = torch.utils.data.Subset(dataset_test, indices[-50:])

    # define training and validation data loaders
    data_loader = torch.utils.data.DataLoader(
        dataset, batch_size=16, shuffle=True, num_workers=4, collate_fn=utils.collate_fn
    )
    images, labels = next(iter(data_loader))

    data_loader_test = torch.utils.data.DataLoader(
        dataset_test,
        batch_size=1,
        shuffle=False,
        num_workers=4,
        collate_fn=utils.collate_fn,
    )

    device = torch.device("cuda") if torch.cuda.is_available() else torch.device("cpu")
    # change to device
    # move model to the right device
    model.to(device)

    # construct an optimizer

    params = [p for p in model.parameters() if p.requires_grad]
    optimizer = torch.optim.SGD(params, lr=0.001, momentum=0.9, weight_decay=0.0005)

    # and a learning rate scheduler which decreases the learning rate by
    # 10x every 3 epochs
    lr_scheduler = torch.optim.lr_scheduler.StepLR(optimizer, step_size=3, gamma=0.1)

**Comment by fmassa** [comment]
From your training losses, it seems that you might want to change the `step_size` from the `lr_scheduler` to a higher value, as you could potentially get better training values.

Try training for more epochs (like 100) and decrease the learning rate every 30 epochs, and see if it increases the performance.

**Comment by datumbox** [comment]
This issue appears to be stale, so I'm closing it. If you think it should remain open, feel free to reopen it.
